pairing/bn256/point.go

```diff
@@ -62,7 +62,34 @@ func (p *pointG1) Clone() kyber.Point {
 }
 
 func (p *pointG1) EmbedLen() int {
-	panic("bn256.G1: unsupported operation")
+	return (255 - 8 - 8) / 8
+}
+
+func gfpToBig(gfp *gfP) *big.Int {
```

Comment:
Could you move this as a function of `gfP` ? So you can do do `curveB.BigInt()`

Reply:
Good idea! It should be included now.